registry/k8s/replication-controller.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: kube-registry-v0
+  labels:
+    k8s-app: kube-registry
+    version: v0
+    kubernetes.io/cluster-service: "true"
+spec:
+  replicas: 1
+  selector:
+    k8s-app: kube-registry
+    version: v0
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-registry
+        version: v0
+        kubernetes.io/cluster-service: "true"
+    spec:
+      containers:
+      - name: registry
+        image: registry:2
+        resources:
+          # keep request = limit to keep this container in guaranteed class
```

Comment:
According to [doc](http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_resourcerequirements) this is not necessary. If requests is omitted it will be set up to limits.